tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -111,6 +114,19 @@ def is_favorites_list_displayed(self):
         return self.marionette.find_element(*self._favorites_list_locator).is_displayed()
 
     @property
+    def is_ice_list_displayed(self):
+        return self.marionette.find_element(*self._ice_list_icon_locator).is_displayed()
+
+    def open_ice_contact_list(self):
+        ice_list = Wait(self.marionette).until(expected.element_present(*self._ice_list_icon_locator))
+        Wait(self.marionette).until(expected.element_displayed(ice_list))
+        ice_list.tap()
+        return Contacts(self.marionette)
+
+    def wait_for_ice_contact_shown(self):
+        return self.marionette.find_element(*self._ice_contact).is_displayed()
```

Comment:
This should be `Wait(self.marionette).until(expected.element_displayed(*self._ice_contact))`